p4-16/spec/P4-16-spec.mdk

```diff
@@ -4035,37 +4077,32 @@ field access, written using dot (".") notation---e.g., `s.field`. If
 copying `struct`s using assignment when the source and target of the
 assignment have the same type. Finally, `struct`s can be initialized
 with a list expression, as discussed in Section [#sec-list-exprs], or
-with a structure initializer, as described in
-[#sec-structure-initializers].  Both these cases must initialize all
+with a structure-valued expression, as described in
+[#sec-structure-expressions].  Both these cases must initialize all
 fields of the structure.
 
-Two structs can be compared for equality (==) or inequality (!=) only
-if they have the same type and all of their fields can be recursively
-compared for equality.  Two structures are equal if and only if all
-their corresponding fields are equal.
-
-## Structure initializers { #sec-structure-initializers }
```

Comment:
To other reviewers: It is intentional that these changes remove the section "Structure initializers".  The intent is that they should be removed from the spec, since they are redundant with the addition of structure-valued expressions.  @MollyDream  please add another comment replying to this one if I have stated things incorrectly here.

Reply:
That is exactly right Andy. "We merged `8.15. Structure initializers` into `8.12. Structure-valued expressions` and `8.14. Operations on struct types` to reduce confusion and redundancy and to make the spec more organized."